hardhat/tasks/sips/args/sipArgs.js

```diff
@@ -759,6 +759,27 @@ const getArgsSip0047 = async (hre) => {
     return { args, governor: "GovernorOwner" };
 };
 
+const getArgsSipSov625 = async (hre) => {
```

Comment:
pls add onchain SIP creation and execution test

Reply:
Added in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/529/commits/eaf61228f6677387d6ee5b37b08ed83a61ceb269)